### PR TITLE
Change the provider to support TF SDK v2

### PR DESCRIPTION
**Pull request by bardielle**

### CODE REVIEW

File: build/ci-tf-e2e.Dockerfile [hunk 1]
```diff
@@ -25,6 +25,6 @@ RUN yum install -y yum-utils && \
     go env -w GO111MODULE=on &&\
     go install github.com/onsi/ginkgo/v2/ginkgo@latest &&\
     go install github.com/golang/mock/mockgen@v1.6.0 &&\
-    cd terraform-provider-rhcs && go mod tidy && go mod vendor && make install &&\
+    cd terraform-cprovider-rhcs && go mod tidy && go mod vendor && make install &&\
```
Comment (osherdp) [inline build/ci-tf-e2e.Dockerfile:28]: cprovider?

File: internal/rhcs/cloudprovider/cloud_providers_state.go [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+				Schema: itemAttributes(),
+			},
+			Computed: true,
+		},
+	}
+}
+func itemAttributes() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"id": {
+			Description: "Unique identifier of the cloud clusterservice. This is what " +
```
Comment (osherdp) [inline internal/rhcs/cloudprovider/cloud_providers_state.go:57]: this seems like a really weird replace-all. it should be 
```suggestion
			Description: "Unique identifier of the cloud provider. This is what " +
```

File: ci/apply_folder.sh [hunk 3]
```diff
@@ -65,7 +65,7 @@ export TF_LOG=${TF_LOG:-info}
 cd "${WORK_DIR}"
 
 
-# Check for provider source and version 'provider_source = "hashicorp/rhcs" (or "terraform.local/local/rhcs"') and 'provider_version = ">=1.0.1"'  
+# Check for clusterservice source and version 'provider_source = "hashicorp/rhcs" (or "terraform.local/local/rhcs"') and 'provider_version = ">=1.0.1"'
```
Comment (osherdp) [inline ci/apply_folder.sh:68]: looks like a bad string replacement

**Review by osherdp — CHANGES_REQUESTED** [review]

File: build/ci-tf-e2e.Dockerfile (hunk 1 above)
Comment (osherdp) [reply build/ci-tf-e2e.Dockerfile:28]: handled via https://github.com/terraform-redhat/terraform-provider-rhcs/pull/263

File: ci/apply_folder.sh (hunk 3 above)
Comment (osherdp) [reply ci/apply_folder.sh:68]: handled via https://github.com/terraform-redhat/terraform-provider-rhcs/pull/263

File: internal/rhcs/cloudprovider/cloud_providers_state.go (hunk 2 above)
Comment (osherdp) [reply internal/rhcs/cloudprovider/cloud_providers_state.go:57]: handled via https://github.com/terraform-redhat/terraform-provider-rhcs/pull/263

**Comment by amalykhi** [comment]
ooo. I think the CI fails because of your changes. 
CI changes PR will fail because it would miss your PR)

**Comment by nirarg** [comment]
Closing This PR, because moving to latest framework

**Pull request closed without merging**